rewrite-yaml/src/main/java/org/openrewrite/yaml/MergeYaml.java

```diff
@@ -37,23 +45,23 @@
             example = "labels: \n\tlabel-one: \"value-one\"")
     @Language("yml")
     String yaml;
+    @Option(displayName = "Filter YAML snippet",
+            description = "The YAML snippet to to use as a basis for filtering which files are applicable to this recipe.",
+            example = "labels: \n\tlabel-two: \"value-two\"",
+            required = false)
+    @Language("yml")
+    @Nullable
+    String filterYaml;
```

Comment:
Seems like this shouldn't be `@Nullable` since `validate()` requires that it be not `null` or empty.